tutorials/classical_simulation/Simulating with SPAM errors.ipynb

```diff
@@ -0,0 +1,335 @@
+{
```

Comment:
<p>The more runs we will <em>perform</em>, the closer the maximum population will <strong>be of </strong>0.6.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>